### DISCUSSION TITLE
Webservices Premium

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16920] [General] [answered]
Does webservices premium will work on local network not connected to Internet?

Best

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Yes, the API works as a regular network service, it will also work on local addresses

**Reply by garryatbrighton** [reply]
Marius, if we refer to https://github.com/YetiForceCompany/YetiForceCRM/issues/16994 the doubt raising about possibility of working offline.
I never tried but been asked to set system on local network only (without access to internet). In light of mentioned issue - how it will be possible if Yetiforce in some parts calling to external services?

Best!

**Reply by mariuszkrzaczkowski** [reply]
set it to false
https://doc.yetiforce.com/code/classes/Config-Performance.html#property_ACCESS_TO_INTERNET
